scripts/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+platforms = ["centos7", "debian9", "ubuntu16", "ubuntu18"]
+
+def buildClosures(arg) {
+    println arg.inspect()
+    def platforms = arg
+    def closures = [:]
+    for (value in platforms) {
+        //final valueCopy = value
+
+        closures[value] = {platform ->
+                stage("build-${platform}") {
+                    docker.build("${platform}-master-test","-f ./scripts/docker/build-${platform}/Dockerfile.deps ./scripts/docker/build-${platform}").inside {
+                        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CleanBeforeCheckout'],[$class: 'RelativeTargetDirectory', relativeTargetDir: "${platform}/build"]], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/mattrose/freeradius-server']]])
+                        sh "cat /etc/os-release || cat /etc/redhat-release"
+                        dir("${platform}/build") {
+                            if (platform.contains("centos")) {
+                                sh 'QA_RPATHS=0x0003 RADIUSD_VERSION_STRING=$(cat VERSION) make rpm'
+                                sh "mv rpmbuild/RPMS/x86_64/*.rpm .."
+                            } else {
+                                sh "apt-get install -y unixodbc-dev"
```

Comment:
Is there a reason this is done manually? It should be a normal .deb dependency.

Reply:
Yes, good point.  It might be there's multiple providers I remember there being something weird with odbc and debian packaging.

Reply:
It's kind of a side-effect of how the Dockerfiles were built.  Because the dockerfiles in master try to be version-agnostic, they end up installing some other odbc package that conflicts with unixodbc, so I have to over-ride the package install from the dockerfile.

Reply:
Maybe older versions depended on iodbc instead? But the solution there is possibly just to not install dependencies from the older versions? Well, unless v3 had it I guess.

Reply:
Prior to v4.0.x there were two database drivers, one for iodbc and one of unixodbc.  In v4 we removed iodbc.  We can likely just adjust the dependencies in the packages to not build the iodbc driver in v3 and v2?